.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn commitlint --edit
```

Comment:
This project uses `npm` instead of `yarn` or `pnpm`. Please use `npx`

Reply:
Also, we might need to pass in `$1`

```
npx commitlint --edit $1
```
or
```
pnpm exec commitlint --edit $1
```

Reply:
Roget that